torch/distributed/autograd/__init__.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+if hasattr(torch._C, "_dist_autograd_init") and not torch._C._dist_autograd_init():
+    raise RuntimeError("Failed to initialize PyTorch distributed autograd support")
+
+class context:
```

Comment:
If you're going to omit `context(object)`, it should be a lot more obvious in this file that it is Python 3 only...